flow360/component/simulation/meshing_param/params.py

```diff
@@ -18,23 +19,58 @@ class MeshingParameters(Flow360BaseModel):
-    refinements: Optional[List[Union[EdgeRefinementTypes, FaceRefinement, ZoneRefinementTypes]]]
-        Refinements for meshing.
+    Affects volume meshing:
+    - farfield
+    - refinement_factor
+    - gap_treatment_strength
+    - volume_layer_growth_rate
+    - volume_layer_first_layer_thickness
+    - `class` UniformRefinement
+    - `class` CylindricalRefinement
+
+    Affects surface meshing:
+    - max_edge_length
+    - curvature_resolution_angle
+    - surface_layer_growth_rate
+    - `class` FaceRefinement
+    - `class` AnisoSurfaceEdge
+    - `class` ProjectAnisoSurfaceEdge
     """
 
-    # Global fields:
-    farfield: Optional[Farfield] = pd.Field()
-    refinement_factor: Optional[pd.PositiveFloat] = pd.Field()
-    gap_treatment_strength: Optional[float] = pd.Field(ge=0, le=1)
+    # Volume **defaults**:
+    farfield: Literal["auto", "quasi-3d", "user-defined"] = pd.Field(
+        description="Type of farfield generation."
+    )
+    refinement_factor: pd.PositiveFloat = pd.Field(
+        description="If refinementFactor=r is provided all spacings in refinement regions and first layer thickness will be adjusted to generate r-times finer mesh."
+    )
+    gap_treatment_strength: float = pd.Field(
+        ge=0,
+        le=1,
+        description="Narrow gap treatment strength used when two surfaces are in close proximity. Use a value between 0 and 1, where 0 is no treatment and 1 is the most conservative treatment. This parameter has a global impact where the anisotropic transition into the isotropic mesh. However, the impact on regions without close proximity is negligible.",
+    )
+    volume_layer_growth_rate: pd.PositiveFloat = pd.Field(
+        gt=1, description="Growth rate for volume prism layers."
+    )
+    volume_layer_first_layer_thickness: pd.PositiveFloat = pd.Field(
+        description="Default first layer thickness for volumetric anisotropic layers."
+    )
+
+    # Face **defaults**
+    max_edge_length: LengthType.PositiveFloat = pd.Field(
+        description="Global maximum edge length for surface cells. This value will be overwritten by the local specification if provided."
+    )  # Note: Applied to faces without faceName and therefore has to exist in root level.
+    curvature_resolution_angle: pd.PositiveFloat = pd.Field(
+        description="""
+        Global maximum angular deviation in degrees. This value will restrict:
+        (1) The angle between a cell’s normal and its underlying surface normal
+        (2) The angle between a line segment’s normal and its underlying curve normal
+        """
+    )
+    surface_layer_growth_rate: float = pd.Field(
+        ge=1, description="Growth rate of the anisotropic layers grown from the edges."
+    )
```

Comment:
similarly, AnisoSurfaceEdge could be used here

Reply:
I asked Runda and he says specifying `growth_rate` per edge is not possible in Pointwise. The best pointwise can do is to have `surface_layer_growth_rate` per face.

For this one we can try move into the `AnisoSurfaceEdge` because for edges if they have no EdgeName we just ignore them and the mesher runs fine. So it is ok to have no parameter for unnamed edges.

Reply:
The best pointwise can do is to have `surface_layer_growth_rate` per "domain"

Reply:
Keeping surface_layer_growth_rate as is because it does not make sense to move into a per-face definition.